scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
> Can't we change the tempest config to use the right driver?

That would be nice. I thought we just need to use a different ```--os-distro```  (the image_id is already set in tempest.conf)